Microsoft.Azure.CosmosRepository/src/IRepository.cs

```diff
@@ -180,7 +196,8 @@ public interface IRepository<TItem> where TItem : IItem
         /// <param name="partitionKeyValue">The partition key value if different than the <see cref="IItem.Id"/>.</param>
         /// <param name="cancellationToken">The cancellation token to use when making asynchronous operations.</param>
         /// <returns>A <see cref="ValueTask"/> representing the asynchronous exists operation.</returns>
-        ValueTask<bool> ExistsAsync(string id, string partitionKeyValue = null, CancellationToken cancellationToken = default);
+        ValueTask<bool> ExistsAsync(string id, string partitionKeyValue = null,

```

Comment:
This is just a nit, but if we wrap some - I prefer wrapping all parameter lists.

Reply:
Ah yes I did not spot this will update while I am here.